spec/lending.md

```diff
@@ -0,0 +1,12 @@
+# Lending
+
+Lending is essential to a margin based protocol. In order to trade margin a user must be able to get the added funds required to over leverage themselves.
+There will be three users of the lending protocol.
+
+First, a user who wants to lend out their money. This user will come to the entropy protocol, add funds to a lending pool. The reason a user will want to do this is because there are users who will want to borrow money in order to margin trade.
+
+Second, a user who wants to borrow money. This user will create an account to send money to, this money will represent the collateral for the loan. The c-ratio will be 150%. This means for a loan of $100 the user must put up $150. A user will want to loan money for countless reasons, one being if they believe the underlying collateral token will raise in price.
```

Comment:
Can collateral currency and the currency that is lent be different?

Reply:
What happens to this locked up collateral btw? Are we going to try to utilize it for something

Reply:
I don't think we can use it for anything. It has to be available for liquidation if it goes under the c-ratio.